addons/account/models/chart_template.py

```diff
@@ -606,6 +610,141 @@ def _setup_utility_bank_accounts(self, template_code, company, template_data):
         for company_attr_name, account in zip(accounts_data.keys(), accounts):
             company[company_attr_name] = account
 
+    @api.model
+    def _instantiate_foreign_taxes(self, country, company):
+        """Create and configure foreign taxes from the provided country.
+
+        Instantiate the taxes as they would be for the foreign localization only replacing the accounts used by the most
+        probable account we can retrieve from the company's localization.
+        This method is intended as a shortcut for instantiation, accelerating it, not as an out-of-the-box solution 100%
+        correct solution.
+        """
+        # Implementation:
+        # - Check if there is any tax for this country and stop the process if yes
+        # - Retrieve the tax group and tax template data
+        # - Create accounts at most probable location in the CoA and assign those to fields requiring an account
+        # - Creates tax group and taxes with their ir.model.data
+
+        taxes_in_country = self.env['account.tax'].search([
+            ('country_id', '=', country.id),
+            ('company_id', '=', company.id)
+        ])
+        if taxes_in_country:
+            return
+
+        def create_foreign_tax_account(existing_account, additional_label):
+            new_code = self.env['account.account']._search_new_account_code(
+                existing_account.company_id,
+                len(existing_account.code),
+                existing_account.code[:-2]
+            )
+            return self.env['account.account'].create({
+                'name': f"{existing_account.name} - {additional_label}",
+                'code': new_code,
+                'account_type': existing_account.account_type,
+                'company_id': existing_account.company_id.id,
+            })
+
+        existing_accounts = {'': None, None: None}  # keeps tracks of the created account by foreign xml_id
```

Comment:
What's the point of these values? You only do `.get()` on it anyways

Reply:
You are right, I'm dumb: I made a noob mistake.
I should have checked by using `xml_id in existing_account`.

The reason is that I want to avoid trying to create an account when I don't need to